portfolio/src/main/webapp/style.css

```diff
@@ -1,12 +1,15 @@
 
 #content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+ 
+  padding: 0px;
```

Comment:
- I don't think that `padding`, `margin` and `border` are having any effect (you can remove all three without a change to the page).

Reply:
Fixed!